aspnetcore/blazor/common/samples/5.x/BlazorServerEFCoreSample/BlazorServerDbContextExample/Pages/Index.razor

```diff
@@ -0,0 +1,242 @@
+﻿@page "/"
+@page "/{Page:int}"
+
+@using Microsoft.EntityFrameworkCore
+
+@inject IContactFilters Filters
+@inject IDbContextFactory<ContactContext> DbFactory
+@inject NavigationManager Nav
+@inject GridQueryAdapter QueryAdapter
+
+<h1>Contacts</h1>
+<GridWrapper @ref="Wrapper"
+             FilterChanged="async () => await ReloadAsync()"
```

Comment:
Can this be simplified?

```suggestion
             FilterChanged="ReloadAsync"
```

Reply:
Simplified this throughout the codebase.